packages/@icon-magic/cli/README.md

```diff
@@ -16,29 +16,8 @@ or using yarn:
 yarn add @icon-magic/cli --dev
 ```
 
-## Glossary
```

Comment:
Where is the Glossary if removed from the CLI?

Reply:
It's in the main README under Getting Started, none of the terms in the glossary are used in the readme for CLI (except for asset)